apps/class-solid/src/components/Analysis.tsx

```diff
@@ -288,10 +297,12 @@ export function VerticalProfilePlot({
     })) as ChartData<Point>[];
 
   const allX = () => [
+    ...firePlumes().flatMap((p) => p.data.map((d) => d.x)),
```

Comment:
I think this triggers an error when you try to plot a variable that's not available in the plume data